### PR TITLE
Enable more tests

### PR DESCRIPTION
**Pull request by nodh**
Builds upon https://github.com/a-sit-plus/vck/pull/136, probably resolving https://github.com/a-sit-plus/mobile-driving-licence-credential/issues/6?

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("at.asitplus.gradle.conventions") version "2.0.0+20240725"
+    id("at.asitplus.gradle.conventions") version "2.0.0+20240905"
```
Comment (JesusMcCloud) [inline build.gradle.kts:2]: better update to 2.0.20+20240920

File: mobiledrivinglicence/src/commonTest/kotlin/at/asitplus/wallet/mdl/CborSerializationTest.kt [hunk 2]
```diff
@@ -52,7 +45,7 @@ class CborSerializationTest : FreeSpec({
             unDistinguishingSign = "AT"
         )
 
-        val serialized = mdl.serialize().encodeToString(Base16(strict = true)).uppercase()
+        val serialized = mdl.serialize().encodeToString(Base16(true)).uppercase()
```
Comment (JesusMcCloud) [inline mobiledrivinglicence/src/commonTest/kotlin/at/asitplus/wallet/mdl/CborSerializationTest.kt:48]: it was more expressive with named param, but ok

**Review by JesusMcCloud — COMMENTED** [review]
do we really want that to go into main. this composite build requiring it to be a subdirectory of VC-K? if we didn't need this credential to align perfectly with VC-K for testing, we could just update smoothly whenever needed after a VC-K release…

**Comment by nodh** [comment]
> do we really want that to go into main. this composite build requiring it to be a subdirectory of VC-K? if we didn't need this credential to align perfectly with VC-K for testing, we could just update smoothly whenever needed after a VC-K release…

No ... i've just used your open branch as the base

**Comment by JesusMcCloud** [comment]
pending everything's green, this is now the preparation for the next release. @nodh pull this into the vc-k release as test dependency yourself, when you prepare it. squash and merge at your discretion. this credentials still needs one final commit to update to stable vc-k before it can be released.

**Review by JesusMcCloud — APPROVED** [review]

**Pull request merged**